libsolidity/formal/CHC.h

```diff
@@ -275,12 +270,24 @@ class CHC: public SMTEncoder
 	//@{
 	struct CHCVerificationTarget: VerificationTarget
 	{
-		smtutil::Expression errorId;
+		unsigned errorId;
+		ASTNode const * const errorNode;
+	};
+
+	struct CHCVerificationTargetPlaceholder
+	{
+		smtutil::Expression constraints;
+		smtutil::Expression errorExpression;
+		smtutil::Expression startingBlock;
 	};
 
 	/// Verification targets corresponding to ASTNodes. There can be multiple targets for a single ASTNode,
 	/// e.g., divByZero and Overflow for signed division.
-	std::map<ASTNode const*, std::vector<CHCVerificationTarget>, IdCompare> m_verificationTargets;
+	std::map<ASTNode const*, std::vector<unsigned>, IdCompare> m_functionVerificationPreTargetIds;
+	std::map<unsigned, CHCVerificationTarget> m_verificationPreTargets;
```

Comment:
Please document what this member is used for.
A new entry is created every time an error condition is seen, right? Even if it is seen multiple times

Reply:
Yes, I have documented that for the first member, since these two members are bound together.